src/generic_core/remote-connection.ts

```diff
@@ -198,6 +198,7 @@ import uproxy_core_api = require('../interfaces/uproxy_core_api');
           },
           false
       ).then((endpoint :net.Endpoint) => {
+        log.info('SOCKS proxy listening on %1', endpoint);
```

Comment:
Well, I wasn't even thinking of PII since we only ever bind to localhost. That could change, I guess, although right now I really have no reason to think why this could leak useful information.